dvc/exceptions.py

```diff
@@ -243,14 +243,6 @@ def __init__(self, path):
         super().__init__(f"file '{path}' not owned by user! ")
 
 
-class DvcIgnoreInCollectedDirError(DvcException):
-    def __init__(self, ignore_dirname):
-        super().__init__(
-            ".dvcignore file should not be in collected dir path: "
-            "'{}'".format(ignore_dirname)
-        )
```

Comment:
Temporarily removed this as it was breaking some tests for importing the whole repo for me. Not quite sure we should error out so violently on this too, need to give it another thought.